src/lib/libwasmfs.js

```diff
@@ -111,7 +111,11 @@ FS.init();
         if (!wasmFSPreloadingFlushed) {
           wasmFSPreloadedDirs.push({parentPath: parent, childName: part});
         } else {
-          FS.mkdir(current);
+          try {
+            FS.mkdir(current);
```

Comment:
I'm actually little confused by this fix.   I tried running `wasmfs.test_fs_js_api` and it passed both with or without this change.

Looking into it it looks the repeated calls to  `FS.mkdir` here always seem to succeed.  I modified the code here to add  some extra debugging and I see:

```

mkdir /home/nested1
0
mkdir /home/nested2
0
```

I'm a little confused how you were hitting an error here?   I must be missing something.

Reply:
Hmm even stranger, browser.test_emscripten_overlapped_package seem to pass both before and after this test.

Reply:
Ah I wasn't running `browser.test_emscripten_overlapped_package_wasmfs`.. that one does fail.